driver-core/src/main/com/mongodb/operation/ChangeStreamOperation.java

```diff
@@ -300,11 +334,21 @@ public BsonArray create(final ConnectionDescription description, final SessionCo
                     changeStream.append("allChangesForCluster", BsonBoolean.TRUE);
                 }
 
+                boolean hasResumeSetting = false;
                 if (resumeToken != null) {
+                    hasResumeSetting = true;
                     changeStream.append("resumeAfter", resumeToken);
-                } else if (startAtOperationTime != null) {
+                }
+                if (startAfter != null) {
+                    hasResumeSetting = true;
+                    changeStream.append("startAfter", startAfter);
+                }
+                if (startAtOperationTime != null) {
+                    hasResumeSetting = true;
                     changeStream.append("startAtOperationTime", startAtOperationTime);
```

Comment:
So spec says to send this even if there's a resume token or start after?  It's just user error to set more than one of them on the initial change stream?

Reply:
Yes:

> startAtOperationTime, resumeAfter, and startAfter are all mutually exclusive; if any two are set, the server will return an error. Drivers MUST NOT throw a custom error, and MUST defer to the server error.